tornado/log.py

```diff
@@ -54,18 +54,20 @@
 
 
 def _stderr_supports_color():
-    color = False
-    if hasattr(sys.stderr, 'isatty') and sys.stderr.isatty():
-        if curses:
-            try:
+    try:
+        if hasattr(sys.stderr, 'isatty') and sys.stderr.isatty():
+            if curses:
                 curses.setupterm()
                 if curses.tigetnum("colors") > 0:
-                    color = True
-            except Exception:
-                pass
-        elif colorama:
-            color = True
-    return color
+                    return True
+            elif colorama:
+                if sys.stderr is getattr(colorama, 'wrapped_stderr', object()):
```

Comment:
OK. Well, I'll go ahead and merge this so we can get a fix out the door, and see what happens with tartley/colorama#128.